storybook/theme.js

```diff
@@ -6,6 +6,36 @@ import { create } from '@storybook/theming/create';
 export default create( {
 	base: 'light',
 	brandTitle: 'WordPress Components',
-	brandImage:
-		'https://s.w.org/style/images/about/WordPress-logotype-wmark.png',
+	brandImage: './wp-org-logo.png',
+
+	// Typography
+	fontBase: '"Inter", sans-serif',
```

Comment:
I don't think we load `Inter` in the Storybook page, double-checking with @mirka — that would mean that only folks who already have it installed locally would actually use it, and everyone else would fallback on the local `sans-serif` font

Reply:
Correct 👍

Reply:
Changing the font to Inter only affected the sidebar, and even when I changed it in the `body` nothing changed in the content. Does anybody know why that would be?

Reply:
It looks like Storybook has its own font family stack:

![Screenshot 2024-10-01 at 23 26 40](https://github.com/user-attachments/assets/802c94e9-2c9e-4244-ac57-604d9cb1e4ce)

According to [the docs](https://storybook.js.org/docs/configure/user-interface/theming#theming-docs), the main UI and the docs are themed differently and may need two separate configurations

Reply:
If it's ok with everyone else, we'll leave this change in the content for the next update, along with your suggestions @mattrwalker and @jasmussen?

Reply:
Yep, updating the font stack to system fonts can happen any time, doesn't have to be this PR. So long as we remember to follow up! 🚀